core/services/ardupilot_manager/settings.py

```diff
@@ -88,7 +89,30 @@ def save(self, content: Dict[str, Any]) -> None:
             Path.mkdir(self.settings_path, exist_ok=True)
 
             with open(self.settings_file, "w+") as file:
-                print(f"Updating settings file: {self.settings_file}")
+                logger.info(f"Updating settings file: {self.settings_file}")
                 json.dump(self.root, file, sort_keys=True, indent=4)
         except Exception as error:
-            warn(f"Could not save settings to disk: {error}")
+            logger.warning(f"Could not save settings to disk: {error}")
+
+
+log_config = {
```

Comment:
Where is this being used ?

Reply:
Not being. It was a test and I forgot to remove it.